HELPS/Controllers/UserController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using HELPS.Services;
+using HELPS.Models;
+
+namespace HELPS.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [Route("api/[controller]")] //change route to desired
+    public class UsersController : ControllerBase
+    {
+        private IUserService _userService;
+
+        public UsersController(IUserService userService)
+        {
+            _userService = userService;
+        }
+
+        [AllowAnonymous]
+        [HttpPost("authenticate")]
+        public IActionResult Authenticate([FromBody]User userParam)
+        {
+            var user = _userService.Authenticate(userParam.Username, userParam.Password);
+
+            if (user == null)
+                return BadRequest(new { message = "Username or password is incorrect" });
+
+            return Ok(user);
+        }
+
+        [HttpGet]
+        public IActionResult GetAll()
+        {
```

Comment:
I am assuming this is for testing but definitely should not be included in production